getting-started/events.md

```diff
@@ -80,72 +80,51 @@ Adds an annotation to the last thread. Keep in mind that sending an annotation c
 `annotation_type` - type of the annotation. This field cannot be empty and is customizable, you can have your own annotation types (i.e. we use "rating" type)
 
 
-<!-- ### Filled form
+### Filled form
 
 A filled form is an event containing some data from a form. Here's what it looks like:
 
 ```js
 {
-  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743", // generated by server side
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
   "custom_id": "12312.301231238591134",
-  "order": 4, // generated by server side
+  "order": 4,
   "type": "filled_form",
   "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
-  "timestamp": 1473433500,  // generated by server side
+  "timestamp": 1473433500,
   "properties": {
     // "Properties" object
   }
-  "fields": [
-    // "Field" objects
-  ]
+  "fields": [{
+      "type": "text",
+      "name": "name",
+      "label": "Your name",
+      "required": true,
+      "value": "John Doe"
+    },
+    {
+      "type": "email",
+      "name": "email",
+      "label": "Your email",
+      "required": true,
+      "value": "john.doe@gmail.com"
+    },
+    {
+      "name": "Welcome to chat",
+      "type": "title",
+      "label": "main title",
+    },
+    {
+      "name": "some text to display",
+      "type": "information",
+      "label": "info",
+    }]
 }
 ```
 
-There are 4 different types of field from filled form, you can use one or more field types in a single filled form event.
-
-`text`:
+As you can see there are 4 different types of field from filled form and you can use one or more field types in a single filled form event. Above example can be displayed as here:
```

Comment:
As you can see there are 4 different types of field from filled form and you can use one or more field types in a single filled form event. Above example can be displayed as here: -> As you can see, there are 4 different types of fields from a filled form. You can use one or more field types in a single filled form event. The above example can be displayed as follows: